robottelo/config/base.py

```diff
@@ -638,6 +638,7 @@ def __init__(self, *args, **kwargs):
         self.image_username = None
         self.image_password = None
         self.image_name = None
+        self.ca_cert = None
```

Comment:
1. This change may have consequences - as it's currently marked as required field, we'll need to update our ci automation to correctly pass some cert to properties file. @san7ket or @ntkathole maybe, could you please provide your feedback? We'll need to make sure our infra is ready before merging such PR.
2. If new property was introduced, it should be included (and preferably documented) in `robottelo.properties.sample`. Please update it accordingly.
3. Pls update `def validate()` method to include this field in validation_errors message.

Reply:
I am sorry, I should have cherry-picked the change to master , for 6.3.z I had corrected this in https://github.com/SatelliteQE/robottelo/pull/5906 , See the base.py